app/views/missing/report.php

```diff
@@ -175,9 +175,9 @@
                                         <label for="body_marks">Body marks </label><br>
                                         <textarea id="body_marks" class="form-control" placeholder="Enter Body Marks" name="person_detail[body_marks]"></textarea><br>
                                         <label for="height">Height </label>
-                                        <input id="height" placeholder="Enter Height" class="form-control" name="person_detail[height]" /><br>
+                                        <input id="height" placeholder="Enter Height (in cms)" class="form-control" name="person_detail[height]" type="number" step="0.01" /><br>
                                         <label for="weight">Weight </label>
-                                        <input class="form-control" placeholder="Enter Weight" id="weight" name="person_detail[weight]" /><br>
+                                        <input class="form-control" placeholder="Enter Weight" id="weight" name="person_detail[weight]" type="number" step="0.01" /><br>
```

Comment:
Here also, the min/max.

Also, change the placeholder to reflect the unit, probably 'kg'?